lib/mutant/reporter/cli/printer/config.rb

```diff
@@ -11,13 +11,11 @@ class Config < self
           #
           # @return [undefined]
           #
-          # rubocop:disable AbcSize
-          #
           # @api private
           def run
             info 'Mutant configuration:'
             info 'Matcher:         %s',      object.matcher.inspect
-            info 'Integration:     %s',      object.integration.name
+            info 'Integration:     %s',      object.integration
             info 'Expect Coverage: %0.2f%%', (object.expected_coverage * 100)
```

Comment:
No the parents are not needed. I just find it more clean. This is DSLish code so I outmitted the main parens for the `#info` call. But for the args I wanted to have them present.

Actually I think want to do explicit parens for everywhere. Will do that in a followup pass.